tuannguyensn2001/edu-rails
# [PEAC-880] Feature/register

Jira Ticket:
- https://employmenthero.atlassian.net/browse/PEAC-880

Summary:
- Register user in system

Actions:
- Write API for register user
- Writer unit test to verify 

Testing plan
- Setup environment
- Start server
- Send request to /api/v1/auth/register with username, password, email
- Expect new user registered if email haven't existed in system
- Expect error if user existed
